test/decision-engine/decision-engine.js

```diff
@@ -648,6 +648,7 @@ describe('Engine', () => {
 
     const onlyCases = []
     for (const testCase of testCases) {
+      // eslint-disable-next-line
```

Comment:
This skips error link with ` testCase.only not permitted [Error/no-only-tests/no-only-tests]`

If I try to specify the actual error in the comment I get linting to shout that `Definition for rule 'no-only-tests' was not found. [Error/no-only-tests]` 🤷🏼